src/format.ts

```diff
@@ -154,13 +154,14 @@ function checkFormat(srcFiles: string[], baseArgs: string[], options: Options):
 function getReplacements(fileXML: string): Replacement[] {
   const replacements: Replacement[] = [];
 
-  let xmlLines = fileXML.split('\n');
-  // The first and last two elements in xmlLines are not needed
-  xmlLines = xmlLines.slice(1, xmlLines.length - 2);
+  let xmlLines = fileXML.trim().split('\n');
+  // first and last elements are outer 'replacements' tags
+  xmlLines = xmlLines.slice(1, xmlLines.length - 1);
 
   for (let i = 0; i < xmlLines.length; i++) {
     // Uses regex to capture the xml attribute and element
-    // XML format: <replacement offset='OFFSET' length='LENGTH'>FIX</replacement
+    // XML format: <replacement offset='OFFSET'
```

Comment:
This looks like it was "fixed" with npm run fix. Can you put the `<...>` on its own line so it doesn't get broken up?